src/helpers/api.ts

```diff
@@ -1,13 +1,11 @@
 import {DATADOG_SITE_US1} from '../constants'
 
+export const datadogSite = process.env.DATADOG_SITE || process.env.DD_SITE || DATADOG_SITE_US1
```

Comment:
I wonder if this should be uppercase, since it's a constant widely used?

```suggestion
export const DATADOG_SITE = process.env.DATADOG_SITE || process.env.DD_SITE || DATADOG_SITE_US1
```

Reply:
Since we have `getBaseIntakeUrl()` in the file, I would prefer to have a `getDatadogSite()` function instead. Otherwise it may not be clear that this "constant" is tied to an environment _variable_